src/main/java/uk/gov/companieshouse/overseasentitiesapi/mapper/TrustDataMapper.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.companieshouse.overseasentitiesapi.mapper;
+
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.overseasentitiesapi.exception.ServiceException;
+import uk.gov.companieshouse.overseasentitiesapi.model.dao.trust.TrustDataDao;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.trust.TrustDataDto;
+
+@Component
+public interface TrustDataMapper {
```

Comment:
Just for consistency with the other mapper we have already, would suggest renaming this to `TrustDtoDaoMapper`.

I presume that the required implementation was too tricky to make use of the 'MapStruct' library?